types/ui-content.ts

```diff
@@ -55,6 +56,7 @@ export type UiContent = {
   location?: string;
   mission: string;
   navigation: MenuContent[];
+  profileMenu?: MenuContent[];
```

Comment:
duplicate de profileMenu, 
reminder:
"is this necessary to add profileMenu here? I could very well all be in uiContent couldn't it? I don't wan't it to be optional and I wan't it to be static for every client for now, so create a defaultProfileMenu with your current fields in a new types file called "static-ui-content.ts", type it correctly and import it we're you need in the components instead of fetching it with the rest of the data"

Reply:
i dont really understand, we will talk about it tomorrow to be sure i understand it well, like i mention earlier